tests/test_vectorestore.py

```diff
@@ -328,3 +328,118 @@ def test_search_from_retriever_interface_with_filter() -> None:
     assert output == [Document(page_content="bar", metadata={"page": "1"})]
 
     docsearch.drop()
+
+
+def test_batch_insertion() -> None:
```

Comment:
This test could be more compact with pytest parametrize fixture. Let's define N as a parameter, fill texts[:N] and check that actual filled size is N. It is enough.